images/fcos-bastion-image/root/usr/bin/clean-registries.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x
+
+mapfile -t ports < <(systemctl list-units 'registry@*' --no-pager --quiet | awk -F'[@.]' '{print $2}')
+for port in "${ports[@]}"; do
+  disk_use=$(df /opt/registry-"${port}" --output='pcent' | grep -o '[0-9]*')
+  if [ "$disk_use" -gt 85 ]; then
+    rm -rf /opt/registry-"${port}"/data/docker/registry/v2/repositories/*
```

Comment:
something is missing here, check the original script as after running the script:

```
/opt/mirror-registry/registry-5000/data/docker/registry/v2/blobs
openshift-qe-bastion blobs # du -sh .
139G	.
openshift-qe-bastion blobs # 

```

Reply:
@aleskandro The script is deleting /data/docker/registry/v2/repositories/* only. 
Deleting /data/docker/registry/v2/blobs/* is missing!

Reply:
What about adding:
rm -rf /opt/registry-"${port}"/data/docker/registry/v2/blobs/*